### PR TITLE
Spelling Updates Batch 1

### PR DESCRIPTION
**Pull request by kingthorin**
Cherry picked from: https://github.com/zaproxy/zap-extensions/pull/3083 and rebased current.

### CODE REVIEW

File: addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties [hunk 1]
```diff
@@ -70,7 +70,7 @@ alertFilters.dialog.add.label.scope.global		= Global
 alertFilters.dialog.button.apply	= Apply
 alertFilters.dialog.button.test		= Test
 alertFilters.dialog.filter.state.notapplied = Not applied
-alertFilters.dialog.filter.state.nottested = Not tested
+alertFilters.dialog.filter.state.untested = Not tested
```
Comment (thc202) [inline addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties:73]: What's the problem here?

Comment (kingthorin) [reply addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties:73]: They're both fine, probably more of a style choice.

Comment (thc202) [reply addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties:73]: But why change the key and not the text? Better just fix typos IMO.

**Comment by thc202** [comment]
Thank you both!

**Review by psiinon — APPROVED** [review]

File: addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties (hunk 1 above)
Comment (jsoref) [reply addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties]: Fwiw, `untested` is a word whereas `not tested` is really two words that one would naturally include some punctuation between.

Note that I'm not attached to the change, but that's the reason for proposing it.

Comment (thc202) [reply addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties]: Like `notapplied` (and a thousand more), which was not changed (not asking for it either).

**Pull request merged**